### PR TITLE
Iset conll feat

### LINKED ISSUE #11: get_iset_conll_feat returns other=HASH(0x30ae3c40)
`Write::CoNLLX` uses `$anode->get_iset_conll_feat` to print the `feat` column (if `feat_attribute eq 'iset'`). Interset can contain feature `other` which is a hash reference and [the current code](https://github.com/ufal/treex/blob/08549e4210c0432b32f7375ff9e13a0974168bc3/lib/Treex/Core/Node/Interset.pm#L237-L252) prints the hash reference address, e.g. 'HASH(0x30ae3c40)', which is probably not the intended behavior.

Simple fix would be to skip `other` and `tagset` feature (as it is done in [the `get_iset_values` method](https://github.com/ufal/treex/blob/08549e4210c0432b32f7375ff9e13a0974168bc3/lib/Treex/Core/Node/Interset.pm#L205-L212)).
I think a better fix would be to delete these methods completely and use `$anode->iset->some_serialization_method`. Unfortunately, I don't see any suitable method in [Lingua::Interset::FeatureStructure](https://metacpan.org/pod/Lingua::Interset::FeatureStructure). If it is missing on purpose, I'll do the 'simple fix' above.

### PR DESCRIPTION
**Pull request by dan-zeman**
The commits in this branch fix #11 .
Interset must be upgraded to 2.047 first.

### CODE REVIEW

File: lib/Treex/Block/A2T/SetGrammatemes.pm [hunk 1]
```diff
@@ -93,14 +93,17 @@ sub process_tnode {
 
 sub set_grammatemes_from_iset {
     my ( $self, $tnode, $anode ) = @_;
-    mapp {
-        my ($name, $value) = ($a, $b);
+    my $isethash = $anode->iset()->get_hash();
+    map {
```
Comment (martinpopel) [inline lib/Treex/Block/A2T/SetGrammatemes.pm]: `foreach` should be used here instead of `map` when the result is not needed

Comment (dan-zeman) [inline lib/Treex/Block/A2T/SetGrammatemes.pm]: OK. Corrected, committed and pushed. It seems that the pull request knows about the change and got updated.

File: lib/Treex/Core/Node/Interset.pm [hunk 2]
```diff
@@ -170,7 +172,7 @@ sub get_iset_structure
 {
     my $self = shift;
     my %f;
-    foreach my $feature ( Lingua::Interset::FeatureStructure::known_features() )
+    foreach my $feature ( $self->iset()->get_nonempty_features() )
```
Comment (martinpopel) [inline lib/Treex/Core/Node/Interset.pm]: t-nodes do not have `iset` attribute, but `dset`.
In general `$self->$interset_attribute->get_nonempty_features()` should be used.

**Comment by martinpopel** [comment]
Travis (this is the red cross and "The Travis CI build failed" message) revealed a bug in this PR, great. See https://travis-ci.org/ufal/treex/builds/78100522 and my second in-code comment, which should fix the bug.

**Pull request merged**